src/app/providers/router/routeConfig/appRouteConfig.tsx

```diff
@@ -0,0 +1,53 @@
+import { FeedbackPage } from 'pages/feedback';
+import { AppRoutesProps } from './types';
+import { getFeedback, getQuestions, getMain, getNotFound, getResult, getCompletedSurvey } from 'app/providers/router/routeConfig/routes';
```

Comment:
в рамках модуля можно и даже нужно использовать относительный путь

```suggestion
import { getFeedback, getQuestions, getMain, getNotFound, getResult, getCompletedSurvey } from '../routes';
```